jenkins-admin/managed-config-files/custom/clouds-yaml/customizedCredentialMappings.sandbox.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
This file should be a symlink of the production file since they will be looking the same way